sysroot/usr/include/kernel/boot/Handoff.hpp

```diff
@@ -145,22 +91,23 @@ class FramebufferInfo {
 class Handoff {
 public:
     // Constructors
+    Handoff();
     Handoff(void* handoff, uint32_t magic);
     ~Handoff();
     // Getters
-    const char* getCmdLine()                { return _cmdline; }
-    const void* getHandle()                 { return _handle; }
-    FramebufferInfo getFramebufferInfo()    { return _fbInfo; }
-    HandoffBootloaderType getBootType()     { return _bootType; }
+    const char* getCmdLine()                    { return _cmdline; }
+    const void* getHandle()                     { return _handle; }
+    fb::FramebufferInfo getFramebufferInfo()    { return _fbInfo; }
+    HandoffBootloaderType getBootType()         { return _bootType; }
 
 private:
     static void parseStivale2(Handoff* that, void* handoff);
     static void parseMultiboot2(Handoff* that, void* handoff);
 
-    const void* _handle;
-    const char* _cmdline;
```

Comment:
Why can't this be a `const char *`? The getter returns a `const char *` already and it's never mutated at any point in the `Handoff`'s lifetime.

Reply:
You cannot use the compiler generated `=` operation if you have const members. We would have to define and implement our own `=` operator if we want to keep it, and that's not worth it.

Reply:
Oh goodness